src/components/patch/PatchContainer.tsx

```diff
@@ -41,24 +62,68 @@ export class PatchContainer extends React.Component<Props, State> {
     const Patches = () => (
       <Grid className="patch-container" container={true} spacing={3}>
         {this.state.visiblePatches.map(patchObj => (
-          <Grid item={true} xs={12} key={patchObj.Version.id}>
-            <Patch Patch={patchObj} client={this.props.client} updateOpenPatches={this.updateOpenPatches} expanded={this.isExpanded(patchObj)} />
+          <Grid item={true} xs={12} key={patchObj.Patch.Id}>
+            <Patch patch={patchObj} builds={this.state.versionsMap[patchObj.Patch.Id].Builds}
+              client={this.props.client} updateOpenPatches={this.updateOpenPatches} expanded={this.isExpanded(patchObj)} />
           </Grid>
         ))}
       </Grid>
     );
 
     return (
       <div>
-        <Banner client={this.props.client} message={"Welcome to the new patches page!"} showOptOut={true} onFinishStateUpdate={null} storageKey={"shouldHideBanner"}/>
+        <Banner client={this.props.client} message={"Welcome to the new patches page!"} showOptOut={true}
+          onFinishStateUpdate={null} storageKey={"shouldHideBanner"} />
         <div className="search-container">
-          <Paper className="search-input" >
-            <InputBase startAdornment={<SearchIcon />}
-              fullWidth={true}
-              placeholder="Search Patch Descriptions"
-              onChange={this.search}
-            />
-          </Paper>
+          <ExpansionPanel>
+            <ExpansionPanelSummary className="search-input">
+              <InputBase startAdornment={<SearchIcon className="search-icon" />}
+                fullWidth={true}
+                placeholder="Search Patch Descriptions"
+                onChange={this.search}
+              />
+            </ExpansionPanelSummary>
+            <ExpansionPanelDetails className="advanced-search">
+              <div>
+                <FormControl className="advanced-select">
+                  <InputLabel>Status</InputLabel>
+                  <Select
```

Comment:
I was thinking that these filters would be independent of the search (ie. you can filter based on status without typing in the search bar). Right now changing this doesn't call the search. What do you think? Also the placement of these suggests that they work in combination with the search, but I kind of think they shouldn't

Reply:
I think you and I thought this ticket meant different things. I was under the impression that the intent here was for these filters to work as a kind of "advanced search," such that changing them would only affect what is displayed while searching. They could certainly be independent of search, it would be super easy to position them somewhere else and use them to filter the list of visible patches. I agree that would make more sense anyways.